internal/commands/app/diff.go

```diff
@@ -44,7 +46,7 @@ func (cmd *CommandDiff) Flags() []flags.Flag {
 		flags.StringFlag{
 			Value: &cmd.inputs.LocalPath,
 			Meta: flags.Meta{
-				Name: "local",
+				Name: flags.FlagLocalPath,
```

Comment:
[nit] instead of centralizing these flag names in the `utils/flags` package, let's keep this defined as `flagLocalPath` in `commands/app/flags.go`